yt/visualization/plot_window.py

```diff
@@ -187,6 +190,13 @@ def __init__(self, data_source, bounds, buff_size=(800,800), antialias=True,
         self.antialias = antialias
         self._axes_unit_names = None
 
+        axname = data_source.ds.coordinates.axis_name[data_source.axis]
+
+        xform = data_source.ds.coordinates.data_transform[axname]
```

Comment:
Just a minor nit -- you mostly refer to transform all spelled out, so seeing it here abbreviated took me a second.

Reply:
Ok I fixed this in b5c2d96e8016efa4d251755874cbe1e82a26f702.